src/utils/tracking.ts

```diff
@@ -1,10 +1,32 @@
+import { CookieStorage } from 'cookie-storage'
+import * as merge from 'deepmerge'
+
+const cookie = new CookieStorage()
```

Comment:
this'll only work client-side though. Is there any time we may want tracking on the server or no?

Reply:
I see your point but I'm not sure if we ever want to do 'this kind' of tracking from SSR. What do you think?

Reply:
lets leave it for now 👍 obviously it wont work at all so i think its ok